lighty-core/dependency-versions/pom.xml

```diff
@@ -25,7 +25,7 @@
             <dependency>
                 <groupId>org.opendaylight.odlparent</groupId>
                 <artifactId>odlparent</artifactId>
-                <version>6.0.11</version>
+                <version>6.0.12</version>
```

Comment:
pick 6.0.13 for https://github.com/opendaylight/odlparent/blob/6.0.x/docs/NEWS.rst#version-6013

Reply:
Done, also I have updated checkstyle to 6.0.13